### PR TITLE
add markdown config to docs

### LINKED ISSUE #210: Suggestion: add markdown rendering for readthedocs
See https://github.com/dib-lab/sourmash/blob/master/doc/conf.py#L295 for an example of using in this in our project.

You will also need to install the `recommonmark` package.

### PR DESCRIPTION
**Pull request by ctb**
Fixes #210.

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -171,3 +171,14 @@
      author, 'BinderHub', 'One line description of project.',
      'Miscellaneous'),
 ]
+
```
Comment (willingc) [inline doc/conf.py]: Please move this higher up in the file. Ideally around line 58.

**Comment by willingc** [comment]
Thanks @ctb. Let's keep this consistent in location in conf.py as we have in other Jupyter docs. See jupyterhub for example if needed.

File: setup.py [hunk 2]
```diff
@@ -9,7 +9,8 @@
         'traitlets',
         'docker',
         'jinja2',
-        'prometheus_client'
+        'prometheus_client',
+        'recommonmark'
```
Comment (yuvipanda) [inline setup.py]: also should we have a different requirements.txt inside the doc folder? binderhub doesn't require recommonmark to run, so am not sure if this belongs here.

What's the general convention for packages needed only for generating docs?

Comment (willingc) [reply setup.py]: I typically put packages needed for docs only in a requirements-doc.txt or environment.yml within the docs directory. See jupyterhub repo for more guidance.

**Comment by choldgraf** [comment]
I'm +1 on this as well once @yuvipanda + @willingc comments are addressed

**Comment by ctb** [comment]
ready for review again, @willingc!

**Comment by yuvipanda** [comment]
LGTM, but I'll wait for @willingc to be back before merging!

**Comment by willingc** [comment]
I'll be back on Thursday :-)

We may want to play around with this (https://github.com/miyakogi/m2r) at some point too.

**Comment by betatim** [comment]
> We may want to play around with this (https://github.com/miyakogi/m2r) at some point too.

I've used pandoc (I think) to convert markdown READMEs to rst. Mostly works but almost always it needed some hand crafting afterwards. It didn't generate syntax errors but the output wasn't quite the same as the original markdown output.

**Comment by willingc** [comment]
@betatim Agree that pandoc isn’t an elegant solution. In my quick scan the other day m2r takes into more account the details of rst, sphinx, docutils.

**Comment by willingc** [comment]
@yuvipanda Thanks for the ping. Made one small change and merged.

**Pull request merged**